dir.props

```diff
@@ -17,6 +17,8 @@
     <DotNetCliToolDir>$(ProjectDir)Tools/dotnetcli/</DotNetCliToolDir>
     <PatchesDir>$(ProjectDir)patches/</PatchesDir>
     <PackagesDir>$(ProjectDir)packages/</PackagesDir>
+    <DotNetSdkDir>$(DotNetCliToolDir)sdk/$(SDK_VERSION)/</DotNetSdkDir>
```

Comment:
I'm not too key on depending on the env variable here it does mean that it will only work when you call build.sh it won't even work for build.cmd.

Reply:
Hmm, yeah, I should have just added a `/p`. Maybe there's even a property that MSBuild sets itself with the location of the SDK that I could have used, didn't look for that either.